org.winehq.Wine.yml

```diff
@@ -25,6 +25,7 @@ finish-args:
   - --env=WINEDLLPATH=/app/vkd3d/lib32:/app/vkd3d/lib:/app/dxvk/lib32:/app/dxvk/lib:/app/lib32/wine/wined3d:/app/lib/wine/wined3d
   - --env=WINEPREFIX=/var/data/proton_pfx
   - --env=GST_PLUGIN_SYSTEM_PATH=/app/lib32/gstreamer-1.0:/app/lib/gstreamer-1.0:/usr/lib/i386-linux-gnu/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0
+  - --env=WINEESYNC=1
```

Comment:
Please put it below `WINEPREFIX`, so that all Wine env vars are grouped together.